digitalearthau/config/products/wofs_annual_summary.yaml

```diff
@@ -0,0 +1,37 @@
+name: wofs_annual_summary
+description: Water Observations from Space Annual Statistics
+metadata_type: eo
+
+metadata:
+  product_type: wofs_annual_summary
+  format:
+    name: 'NetCDF'
+  instrument:
+    name: 'TM,ETM,OLI'
+  platform:
+    code: 'LANDSAT_5,LANDSAT_7,LANDSAT_8'
```

Comment:
The `metadata` field is used for matching dataset metadata with a product.
If in the future we add `LANDSAT_9` to this list in a dataset, it won't pick it up.